inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/config/InstrumentationConfigurationChangedEvent.java

```diff
@@ -0,0 +1,33 @@
+package rocks.inspectit.oce.core.instrumentation.config;
+
+import lombok.Getter;
+import org.springframework.context.ApplicationEvent;
+
+/**
+ * This event is fired when the {@link InstrumentationConfigurationResolver} has resolved a new {@link InstrumentationConfiguration}.
+ * This event usually occurs after a {@link rocks.inspectit.oce.core.config.InspectitConfigChangedEvent}.
+ * The new {@link rocks.inspectit.oce.core.config.model.InspectitConfig} is used to derive the {@link InstrumentationConfiguration},
+ * for which then this event is triggered.
+ */
+
+public class InstrumentationConfigurationChangedEvent extends ApplicationEvent {
+
+    /**
+     * The configuration which has previously been active.
+     */
+    @Getter
```

Comment:
``@Getter`` would also work on a class level, then you don't have to add it to all members.